components/automate-chef-io/content/docs/client-runs.md

```diff
@@ -216,4 +233,5 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/delete-nodes/config"
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests. 
+You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.

```

Comment:
```suggestion
```

Reply:
seems like we shouldnt be keeping this a1 content around, i added the corrected link to the line above